### ISSUE TITLE
Support import of patch series

### ISSUE THREAD

**Issue by joshtriplett** [git-series/git-series#32]
Add a command to import a patch series from a series of files, and make it the next version of a series (or a new series).  In addition to simplifying the process of turning a `format-patch`ed series into a git-series commit, this would also enable advanced workflows like "export series as patches, run sed, re-import series, git series commit".

**Comment by danvet** [comment]
What about making import/export complete inverse functions of each another? I.e. cover letter (also with sub series, issue #33) and maybe unapplied patches (e.g. when importing a patch series that doesn't apply cleanly a la quilt push/pop) should not get lost. For all those we might need a git series export, which also writes a series file. And add a --series option or similar to import for that additional data.

**Comment by joshtriplett** [comment]
@danvet Agreed.  Any git-series metadata should round-trip through import/export.

I don't think I need a series file quite yet; I can rely on format-patch's numbered files instead.  Quilt import/export would be a separate issue from git-series native import/export.

**Comment by mherbert** [comment]
Tools like gerrit and github already have pretty good abilities to compare different revisions of a patchset and review the "history of the history"; always had and keep getting better: https://bugs.chromium.org/p/gerrit/issues/detail?id=217 (hide rebase diffs)

Like it or not Gerrit also made it much easier to delete reviews recently: https://groups.google.com/a/chromium.org/d/msg/chromium-os-dev/Q1eeCtK6oEs/Vp3JhUXhBgAJ

For these reasons I think import/export from/to Gerrit/github would be very useful. Does this call for another "issue" or is this issue 32 enough to cover that as well?

**Comment by cz172638** [comment]
what about integreation with git-review, git-pw (patchwork)?
